template/src/project_name/accounts/tests.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+
+class UsersManagersTests(TestCase):
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏You do not need to extend `TestCase`

We are using Pytest for our test suite - not Django's built in unit test framework. You should not extend the Django TestCase. You don't even need to wrap your tests in a class. As long as the test function begins `def test_` it will be picked up.

See: https://github.com/ackama/django-template/blob/main/template/src/tests/system/main/test_views.py for an example.